### PR TITLE
Feature/explore page

### PR DESCRIPTION
**Pull request by clementprdhomme**
This PR makes the Explore page looks like the prototype and brings back all of its features.

### CODE REVIEW

**Comment by clementprdhomme** [comment]
😱 I forgot to tell you, you have to `npm install` again.

File: components/app/explore/DatasetWidget.js [hunk 1]
```diff
@@ -112,14 +80,52 @@ class DatasetWidget extends React.Component {
...
+    if (this.state.hasWidget) {
+      return {
+        ...this.state.widget.attributes,
+        ...{ id: this.state.widget.id }
+      };
+    }
+    if (this.state.hasLayer) {
+      return {
+        ...this.state.layer.attributes,
+        ...{ id: this.state.layer.id }
```
Comment (pablopareja) [inline components/app/explore/DatasetWidget.js]: Why not just putting `id: this.state.layer.id` instead?

Comment (clementprdhomme) [inline components/app/explore/DatasetWidget.js]: Not my code, by of course, this makes sense. When I fixed the eslint issues I didn't change the logic.

**Comment by clementprdhomme** [comment]
I've made the change you asked for!

**Pull request merged**